pkg/plugins/golang/v4/scaffolds/init.go

```diff
@@ -161,6 +162,9 @@ func (s *initScaffolder) Scaffold() error {
 		&templates.Golangci{},
 		&e2e.Test{},
 		&e2e.SuiteTest{},
+		&github.E2eTestCi{},
+		&github.TestCi{},
```

Comment:
@camilamacedo86 Everything looks good to me. Just one question: Is it necessary to maintain separate files for each of GH Action template?

Reply:
We want to set up 3 separate jobs as follows:

<img width="684" alt="Screenshot 2024-09-03 at 18 53 41" src="https://github.com/user-attachments/assets/822fa446-e294-4cbd-ba33-ce17b95dccd6">


Each job will have its own file so that if a failure occurs, it's easy to identify which target/job/test failed. Running the jobs in parallel will also make the process faster and easier to manage.

This separation also makes it clear which file corresponds to each task: test.yml for tests, lint.yml for linting, and so on. This approach simplifies any future changes and helps maintain clarity—if everything were in the same file, it would be more challenging to pinpoint issues and make updates.

Reply:
@camilamacedo86  makes sense to maintain the separation of concern. Everything else seems good for now.